eth/protocols/eth/protocol.go

```diff
@@ -41,7 +41,7 @@ const ProtocolName = "eth"
 
 // ProtocolVersions are the supported versions of the `eth` protocol (first
 // is primary).
-var ProtocolVersions = []uint{ETH68, ETH67, ETH66}
+var ProtocolVersions = []uint{ETH67, ETH68, ETH66}
```

Comment:
Ah, any particular reason for this change? A unintentional mistake?

Reply:
yes